packages/neos-ui-editors/src/Library/index.js

```diff
@@ -0,0 +1,5 @@
+import AssetUpload from './AssetUpload';
+
+export default () => ({
+    AssetUpload
+});
```

Comment:
This is wrong; you need to do an "export AssetUpload" here :)

A named import in the form `import {....}` is *never* accessing the default import; they are two totally different things.